server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -350,15 +352,15 @@ public static GetResult fromXContentEmbedded(XContentParser parser, String index
                     }
                 } else if (FIELDS.equals(currentFieldName)) {
                     while(parser.nextToken() != XContentParser.Token.END_OBJECT) {
-                        DocumentField getField = DocumentField.fromXContent(parser);
+                        DocumentField getField = DocumentField.fromXContent(parser, false);
```

Comment:
@rjernst This DocumentField is not inside of `_source`, but I assumed that it is not a metadata field based on the way the FIELDS component is filled during serialization https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/index/get/GetResult.java#L277-L282